src/table/table.scss

```diff
@@ -16,75 +15,205 @@
-    background-color: transparent;
-    border: none;
+@mixin iui-table-header {
+  width: 100%;
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
   }
 
-  .iui-tables-row {
-    margin-top: -1px;
+  .iui-row {
     display: flex;
     flex-grow: 1;
     width: 100%;
-    border-top: 1px solid transparent;
-    border-left: 1px solid transparent;
-    border-right: 1px solid transparent;
+  }
+
+  .iui-cell:not(.iui-slot) {
+    @include iui-focus;
+    cursor: pointer;
+    position: relative;
+    overflow: visible;
+
+    > .iui-borderless {
+      margin-left: $iui-xs;
+
+      &:not(.iui-active) {
+        visibility: hidden;
+      }
+    }
+
+    > .iui-resizer {
+      height: 100%;
+      width: $iui-m;
+      position: absolute;
+      top: 0;
+      right: 0;
+      transform: translateX(50%);
+      touch-action: none;
+      cursor: ew-resize;
+      z-index: 100;
+      display: none;
+        
+      > .iui-resizer-bar {
+        height: 100%;
+        width: $iui-xxs;
+        margin: 0 auto;
+        transition: background-color $iui-speed-fast ease-out, width $iui-speed-fast ease-out;
+        @include themed {
+          background-color: t(iui-color-background-5);
+        }
+      }
+
+      &:hover > .iui-resizer-bar {
+        width: $iui-xs;
+        @include themed {
+          background-color: t(iui-color-foreground-primary);
+        }
+      }
+    }
+
+    &:hover,
+    &:focus,
+    &:focus-within {
+      @include themed {
+        background-color: t(iui-color-background-4);
+      }
 
+      .iui-sort,
+      .iui-borderless {
+        visibility: visible;
+      }
+    }
+
+    &:hover {
+      > .iui-resizer {
+        display: block;
+      }
+    }
+  }
+
+  // Sort icon
+  .iui-sort {
+    visibility: hidden;
+    margin-left: auto;
+    width: $iui-icons-default;
+    height: $iui-icons-default;
     @include themed {
-      border-bottom: solid 1px t(iui-color-background-4);
+      fill: t(iui-icons-color-actionable);
     }
+  }
 
-    > .iui-tables-slot > .iui-tables-menu {
-      visibility: hidden;
+  // Sorted column
+  .iui-sorted {
+    @include themed {
+      background-color: t(iui-color-background-4);
+    }
+
+    .iui-sort {
+      visibility: visible;
+        
+      &,
+      > * {
+        @include themed {
+          fill: t(iui-icons-color-actionable);
+        }
+      }
+    }
+  }
+
+  // Double header
+  .iui-row + .iui-row {
+    @include themed {
+      border-top: 1px solid t(iui-color-background-5);
     }
+  }
+}
+
+@mixin iui-table-body {
+  width: 100%;
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+  
+  .iui-row {
+    display: flex;
+    flex-grow: 1;
+    width: 100%;
+    margin-top: -1px;
+    border: solid 1px transparent;
 
-    &:hover:not(.iui-tables-row-active, .iui-tables-paginator) {
+    // Last row doesn't get bottom border. Up for debate.
+    &:not(:last-child) {
```

Comment:
For me looks weird.. I would vote last row should get bottom border...

Reply:
I removed the last row's line and instead applied it on the paginator.  The two main reasons I am thinking of right now:
1. The table uses background.1 and in scenarios like the one I presented in UXT, the table is placed above background.2.  In situations like this, the background color difference essentially makes up for the lack of a border.
![Screen Shot 2021-06-01 at 3 10 26 PM](https://user-images.githubusercontent.com/849817/120377729-81758d80-c2eb-11eb-8660-3cf0ce12a71e.png)
![Screen Shot 2021-06-01 at 3 12 46 PM](https://user-images.githubusercontent.com/849817/120377973-d31e1800-c2eb-11eb-8878-1e0038e7eacd.png)
2. I reproduced the basic table / list shown in this GIF from the file upload standard.  It looked weird having a dividing line underneath _MyFile-3.pdf_.
![table-empty](https://user-images.githubusercontent.com/849817/120378203-1d06fe00-c2ec-11eb-94cd-7973e668493f.gif)

Reply:
I do agree in 2nd example there is no border.. But I also think that is not really a table :D
Maybe background was the one which was messing up things.. in our demo htmls, tables look really bad without bottom border...
![image](https://user-images.githubusercontent.com/81580355/120648480-5bcfbc00-c484-11eb-84c9-5cf81ff3377a.png)